db/fixtures/ae_datastore/ManageIQ/Deployment/ContainerProvider/System/StateMachine/Deployment.class/__methods__/add_provider.rb

```diff
@@ -1,21 +1,17 @@
+gem "net-ssh", "=4.0.0.alpha4"
 require 'net/ssh'
 
 DEPLOYMENT_TYPES = {
-  :origin            => "ManageIQ::Providers::Openshift::ContainerManager",
-  :enterprise        => "ManageIQ::Providers::OpenshiftEnterprise::ContainerManager",
-  :atomic            => "ManageIQ::Providers::Atomic::ContainerManager",
-  :atomic_enterprise => "ManageIQ::Providers::AtomicEnterprise::ContainerManager"
+  :origin                 => "ManageIQ::Providers::Openshift::ContainerManager",
+  :"openshift-enterprise" => "ManageIQ::Providers::OpenshiftEnterprise::ContainerManager",
```

Comment:
I thought maybe you forgot to delete the quotes when switching between a symbol and strings.

So a better question is: Why are you using a dash for `openshift-enterprise` but underscore for `atomic_enterprise`?  Should this be `openshift_enterprise` for consistence and then you do not need the quotes.